bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/gemini/igrins2/Igrins2.scala

```diff
@@ -242,6 +243,9 @@ object Igrins2 {
     SLIT_VIEWING_PROP
   ).map(p => (p.getName, p))
 
+  val getInstConfigInfo: JList[InstConfigInfo] =
+    List[InstConfigInfo]().asJava
```

Comment:
A placeholder for now?

Reply:
The requirement is to not have any controls on the browser yet